<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebuggerForm</class>
 <widget class="QWidget" name="DebuggerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="minimumSize">
      <size>
       <width>441</width>
       <height>411</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabVar">
      <attribute name="title">
       <string>Переменные</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QTreeWidget" name="treeWidgetMain">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>3</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="autoExpandDelay">
          <number>-1</number>
         </property>
         <attribute name="headerCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Имя</string>
          </property>
          <property name="font">
           <font>
            <kerning>true</kerning>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>тип данных</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QTreeWidget" name="treeWidgetWatch">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>5</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>имя</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>значение</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>время</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>качество</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>тип</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>комментарий</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabInOut">
      <attribute name="title">
       <string>Входы/выходы</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabMem">
      <attribute name="title">
       <string>Память</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="minimumSize">
      <size>
       <width>101</width>
       <height>91</height>
      </size>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>старт</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>стоп</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Обновить</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
